### ISSUE TITLE
How to create Gremlin queries? 

### ISSUE THREAD

**Issue by peterj** [Azure/azure-cosmos-python#104]
I noticed that the .NET SDK has a ``` CreateGremlinQuery``` method, but the Python SDK doesn't. I am trying to run a simple query against a graph DB (e.g. ```g.V()```) and it's failing error code ```SC1001``` and message ```Syntax error, incorrect syntax near 'g'.```. 

Here's an example of a call I am making: 
```
collection_link='dbs/mydb/colls/mycoll'
db_query='g.V()'
client.QueryDocuments(collection_link, db_query))
```

Am I missing something or is this simply not implemented/included in the Python SDK yet? If latter, any plans/ETA on when this is going to be supported?

**Comment by moderakh** [comment]
@peterj this SDK only provides documentdb api of cosmos db. There is no gremlin api here.

**Comment by peterj** [comment]
Is there another Python SDK I could use to make Gremlin against DocumentDB?

**Comment by moderakh** [comment]
@jayantacs knows more about Gremlin support. @jayantacs  can you please respond.

**Comment by southpolesteve** [comment]
Closing this issue due to age. Active development of our Python SDK has moved to https://github.com/Azure/azure-sdk-for-python. We have also released 4.0 which includes many updates and new features. If this issue still needs our attention, please reopen it in the new repository.
